src/server/pps/server/worker_rc.go

```diff
@@ -118,6 +118,12 @@ func (a *apiServer) workerPodSpec(options *workerOptions, pipelineInfo *pps.Pipe
 	}, {
 		Name:  client.PPSPipelineNameEnv,
 		Value: pipelineInfo.Pipeline.Name,
+	}, {
+		Name:  "POSTGRES_PORT",
```

Comment:
Maybe we should add a note as to why we're zero'ing these out (to override the kube service name env vars)